js/files.js

```diff
@@ -236,14 +246,18 @@ async function createVectorStore(files) {
     const request = {
         name: prefix + ' Vector Store',
         file_ids: files,
-        expiration: 7,
+        expiration: null,
     };
     $('.loader').show();
     const vs_id = await authClient.fetch(url.toString(),
         { method: 'POST', headers: {'Content-Type': 'application/json'}, body: JSON.stringify(request) }).
         then(r => {
             if (!r.ok) {
-                throw new Error ('Create Vector Store failed:' + r.statusText);
+                return r.json().then(e => {
+                    throw new Error(`${e.error}: ${e.message}`);
+                }).catch(() => {
+                    throw new Error ('Create Vector Store failed:' + r.statusText);
```

Comment:
unless this text is programmatically important
```suggestion
                    throw new Error ('Failed to create Vector Store:' + r.statusText);
```